core/statistics/softwareVersion/softwareVersion.go

```diff
@@ -32,8 +34,14 @@ func NewSoftwareVersionChecker(appStatusHandler core.AppStatusHandler) (*Softwar
 		return nil, core.ErrNilAppStatusHandler
 	}
 
-	// check interval will be random in a interval [1hour, 1hour 15minutes]
-	randInterval := time.Duration(rand.Int() % 15)
+	// check interval will be random in a interval [1hour5minutes , 1hour20minutes]
```

Comment:
`// check interval will be a random duration in the interval [1hour5minutes , 1hour20minutes]`

Reply:
done.